app/views/employers/employer_profiles/_county_field.html.erb

```diff
@@ -0,0 +1,17 @@
+<%#= fields_for(@parent_object, :fieldset => false) do |f| %>
```

Comment:
I'd rather not commit commented out code. It only makes things confusing for a future dev.

If it's unneeded, we don't need to add it. If it's needed but non working, it should get resolved.